### DISCUSSION TITLE
Módulo 1 - Duda "trabajar con números"

### DISCUSSION THREAD

**Discussion by TahiliAlv** [LaunchX-InnovaccionVirtual/CursoIntroPython#206] [Q&A] [answered]
Cuando ejecuto el código no me imprime lo que debería, que estoy haciendo mal? solo copie y pegue el código como viene en el ejemplo. Según yo el ejercicio solo es para ver la diferencia entre cuando tienes el numero solo como caracter y cuando es un valor cuando agregas int, eso se ve cambiando la ultima linea de codigo, eso me queda claro, pero entonces porque solo imprime la primera linea y lo demás se queda cargando y cuando lo detengo salen un monton de outputs todos raros
![Screenshot_175](https://user-images.githubusercontent.com/99460744/153727137-899aaa3b-2539-4d41-a6dd-a77ca3a676bd.png)
![Screenshot_174](https://user-images.githubusercontent.com/99460744/153727139-c2b2afdb-1bd4-4321-be27-f5425740ccde.png)

**Comment by Stmark1** [comment, marked as answer]
Hola, lo que creo que sucede es que no le ingresaste los numeros al input, me explico, cuando ejecutas el programa en la parte superior de vscode aparece un rectangulo para ingresar datos. Ingresando los valores deberia funcionar.

El error que te aparece es "KeyboardInterrupt" que hace referencia a que cancelaste la ejecución del programa con una entrada del teclado, normalmente es "ctrl + c"

**Reply by TahiliAlv** [reply]
Muchas gracias! si justamente era lo que pensaba pero ni idea donde tenia que meter los números c': ahora me queda más que claro

**Comment by DanielOe21** [comment]
### Hola @TahiliAlv !

Lo que pasa es que en tu código tienes un ' input' , eso quiere decir que te pide que le ingreses ' algo' al programa, en este caso pues tienes que ponerle números.

El programa se queda en la primera línea debido a que se mantiene 'esperando'  hasta que desde tu teclado le ingreses un número, después el programa convertirá ese número de <string> a <int> , lo guardará en la variable asignada y el programa seguirá para pedirte el siguiente input y después ya saldrá la respuesta.

Espero haberte ayudado. Por favor no olvides marcar tu pregunta como CONTESTADA si así lo consideras.

### Follow me!
<a href="https://github.com/DanielOe-21">
<img alt="GitHub followers" src="https://img.shields.io/github/followers/DanielOe-21?style=social">
</a>

**Reply by TahiliAlv** [reply]
Perfecto! Muchas gracias por el apoyo, ya ingresando los valores salió bien n_n

**Comment by Ingrzs** [comment]
Cuando ejecutas el codigo, checa en la parte de arriba te aparece un mensaje introducir un numero , si es asi introducelo ya que el codigo te lo esta pidiendo, de ahi presionas enter , y luego te mandara un segundo mensaje pidiendote el otro numero y presionas enter en este caso ya te debe arrojar el resultado... Y en este caso creo el error como comento el de arriba tu codigo se rompe ya que no tiene una instrucción en la que le digas si cancelas/ o metes un simbolo/letra

**Reply by TahiliAlv** [reply]
Gracias! si justamente me hacía falta saber donde introducir el valor

**Comment by TahiliAlv** [comment]
Con sus comentarios me di cuenta que ignoré este pequeño cuadro de diálogo, ya ahí introduje los valores, enter y listo. Muchas gracias! 



![Screenshot_177](https://user-images.githubusercontent.com/99460744/153732057-ff7aa95b-aaf7-4086-ad7a-6dc032cde085.png)
![Screenshot_176](https://user-images.githubusercontent.com/99460744/153732058-886dfb62-b7c6-4002-97cc-8301b0ca5516.png)
